tokio/src/macros/join.rs

```diff
@@ -32,6 +32,25 @@ macro_rules! doc {
         ///
         /// [`tokio::spawn`]: crate::spawn
         ///
+        /// # Poll Ordering
```

Comment:
Let's make this a sub-section together with fairness.
```suggestion
        /// ## Poll Ordering
```